app/views/staff/events/show.html.haml

```diff
@@ -47,64 +47,39 @@
                 %td.text-primary
                   %strong Event Staff:
                 %td= event_teammates.count
-              -# %tr
-                -# %td.text-primary
-                -#   %strong Guidelines:
-                -# %td
-                -#   -if event.guidelines.present?
-                -#     = event.guidelines.truncate(150, separator: /\s/)
-                -#   = link_to "Public guidelines", event_path(slug: event.slug)
-                -#   %p= link_to "<i class='fa fa-pencil'></i> Edit Guidelines".html_safe, event_staff_guidelines_path, class: "btn btn-primary"
-
-      -# .widget.widget-table
-      -#   .widget-header
-      -#     %i.fa.fa-calendar
-      -#     %h3 CFP Details
-      -#   .widget-content
-      -#     %table.table.table-striped.table-bordered
-      -#       %tbody
-      -#         %tr
-      -#           %td.text-primary
-      -#             %strong Days Remaining:
-      -#           %td #{event.cfp_days_remaining}
-      -#         %tr
-      -#           %td.text-primary
-      -#             %strong Event Start:
-      -#           %td #{event.start_date.to_s(:month_day_year) unless event.start_date.blank?}
-      -#         %tr
-      -#           %td.text-primary
-      -#             %strong Event End:
-      -#           %td #{event.end_date.to_s(:month_day_year) unless event.end_date.blank?}
-
 
     .col-sm-4
       .widget
         .widget-header
           %i.fa.fa-list-alt
           %h3 CFP Statistics
         .widget-content
-          %ul.list-group
-            %li.list-group-item
-              %strong.text-primary Total:
-              %span.label.label-info #{event.proposals.count}
-            %li.list-group-item
-              %strong.text-primary Total Proposals Reviewed:
-              %span.label.label-info #{event.proposals.rated.count} (#{event.reviewed_percent})
-            %li.list-group-item
-              %strong.text-primary Reviewed Proposals Accepted:
-              %span.label.label-info #{event.proposals.accepted.count}
-            -# %li.list-group-item
-            -#   %strong.text-primary Confirmed:
-            -#   %span.label.label-info #{event.proposals.accepted.confirmed.count} (#{event.confirmed_percent})
-            %li.list-group-item
-              %strong.text-primary Accepted Sessions Scheduled:
-              %span.label.label-info #{event.proposals.scheduled.count} (#{event.scheduled_percent})
-            %li.list-group-item
-              %strong.text-primary Waitlisted Sessions:
-              %span.label.label-info #{event.proposals.waitlisted.count}
-            -# %li.list-group-item
-            -#   %strong.text-primary Confirmed:
-            -#   %span.label.label-info #{event.proposals.waitlisted.confirmed.count} (#{event.waitlisted_percent})
+          %strong.text-primary No CFP Statistics
+          - if (event.state == Event::STATUSES[:draft]) && current_user.organizer_for_event?(current_event)
```

Comment:
Since we also check if someone is an organizer when you're checking draft status this block only shows for organizers when the event is in draft mode.  It should show in draft mode regardless of role.